hexapod_dart/include/hexapod_dart/descriptors.hpp

```diff
@@ -94,11 +104,12 @@ namespace hexapod_dart {
             RotationTraj() {}
 
             template <typename Simu, typename robot>
-            void operator()(Simu& simu, std::shared_ptr<robot> rob, const Eigen::Vector3d& init_pos, const Eigen::Vector3d& init_rot)
+            void operator()(Simu& simu, std::shared_ptr<robot> rob, const Eigen::Vector6d& init_trans)
```

Comment:
Why does `init_trans` reads *init translation* and not *init transformation*? I do mean the second, though..